CHANGELOG.md

```diff
@@ -71,13 +71,24 @@
 
 #### Core
 
+- Fixed an issue where `kong docker-start` would fail if `KONG_PREFIX` was set to
+  a relative path.
+  [#9337](https://github.com/Kong/kong/pull/9337)
+- Fixed an issue with error-handling and process cleanup in `kong start`.
+  [#9337](https://github.com/Kong/kong/pull/9337)
 - Fix issue where external plugins crashing with unhandled exceptions
   would cause high CPU utilization after the automatic restart.
   [#9384](https://github.com/Kong/kong/pull/9384)
 - Fix issue in `header_filter` instrumentation where the span was not
   correctly created.
   [#9434](https://github.com/Kong/kong/pull/9434)
 
+#### Configuration
+
+- A new configuration item (`resty_events_sock_path`) has been added to allow
```

Comment:
This makes little sense to me, why is this config option something the user should know in the first place?

Reply:
OK, I will remove it in changelog.